pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -769,23 +769,153 @@ void registerLightningTensorBackendAgnosticMeasurements(PyClass &pyclass) {
         });
 }
 
+/**
+ * @brief Register observable classes for TensorNet.
+ *
+ * @tparam LightningBackendT
+ * @param m Pybind module
```

Comment:
Would you like to add name to the docstring?

Reply:
Done!